language/vm/src/file_format.rs

```diff
@@ -739,6 +739,23 @@ impl SignatureToken {
         matches!(self, MutableReference(_))
     }
 
+    /// Returns true if the `SignatureToken` can represent a constant (as in representable in
+    /// the constants table).
+    pub fn is_constant(&self) -> bool {
```

Comment:
I find the wording a bit confusing but not a big deal. maybe `is_valid_for_constant` `valid_constant_type`? 

Just that its not about the type itself being a "constant", which is a term that comes up some times in type systems